src/sentry/release_health/duplex.py

```diff
@@ -116,7 +116,10 @@ def relative_change(self) -> Optional[float]:
         if self._results is None:
             return None
         sessions, metrics = self._results
-        return (metrics - sessions) / abs(sessions)
+        if sessions:
```

Comment:
When there isn't any metrics data the relative change is `-{sessions}`. However, when there isn't any sessions data the relative change is `None` (instead of `+{metrics}`?). Is this correct, or is there something I'm missing?

Note that I've assumed that use case to not be happening at the moment, but I'd rather be more defensive about it to prevent incorrect behavior in the future.

Reply:
>However, when there isn't any sessions data the relative change is None

I think this is reasonable behavior

Reply:
When `metrics` is zero, the relative change from `sessions` is -100%, that is correct IMO. When `sessions` is zero, the relative change is theoretically infinity, but I prefer to map that to `None`.

When both are zero it might make more sense to return 0 instead of `None`, but in that case this function would not even be called.